### PR TITLE
Update the template

### PR DESCRIPTION
**Pull request by yevk**
Added the Domain Name resource, with the certificate and the domain as
parameters.

Moved the resources on the designer a bit to look better.

Updated the README to reflect the change.

### CODE REVIEW

File: .altostra/template.json [hunk 1]
```diff
@@ -12,33 +12,78 @@
...
+					]
+				}
+			},
 			"resources": {
+				"DomainName01": {
+					"certificate": {
+						"type": "cert.stored",
+						"uri": "arn:aws:acm:us-east-1:123415165762:certificate/7e2e43a5-272d-4489-b523-70ac94520eb4"
+					},
+					"dnsName": "www.blog-demo.com",
```
Comment (yevk) [inline .altostra/template.json:71]:
```suggestion
```

File: .altostra/template.json [hunk 2]
```diff
@@ -12,33 +12,78 @@
...
+					"paths": [
+						"resources/DomainName01/dnsName"
+					]
+				}
+			},
 			"resources": {
+				"DomainName01": {
+					"certificate": {
+						"type": "cert.stored",
+						"uri": "arn:aws:acm:us-east-1:123415165762:certificate/7e2e43a5-272d-4489-b523-70ac94520eb4"
```
Comment (yevk) [inline .altostra/template.json]:
```suggestion
						"uri": ""
```

File: .altostra/template.json (hunk 1 above)
Comment (yevk) [inline .altostra/template.json]:
```suggestion
					"dnsName": "www.example.com",
```

File: .altostra/template.json (hunk 2 above)
Comment (ShlomiAltostra) [inline .altostra/template.json]:
```suggestion
						"uri": "arn:aws:acm:us-east-1:123456789012:certificate/00000000-0000-0000-0000-000000000000"
```

File: .altostra/template.json (hunk 1 above)
Comment (ShlomiAltostra) [inline .altostra/template.json]:
```suggestion
					"dnsName": "www.example.com",
```

**Review by ShlomiAltostra — CHANGES_REQUESTED** [review]
Lets sanitize some data (even though it is kept in git)

**Review by ShlomiAltostra — APPROVED** [review]

**Pull request merged**